hector_mapping/src/HectorMappingRos.cpp

```diff
@@ -345,7 +345,7 @@ void HectorMappingRos::scanCallback(const sensor_msgs::LaserScan& scan)
       odom_to_base.setIdentity();
     }
     map_to_odom_ = tf::Transform(poseInfoContainer_.getTfTransform() * odom_to_base.inverse());
-    tfB_->sendTransform( tf::StampedTransform (map_to_odom_, scan.header.stamp, p_map_frame_, p_odom_frame_));
+    tfB_->sendTransform( tf::StampedTransform (map_to_odom_, scan.header.stamp + ros::Duration(1.0), p_map_frame_, p_odom_frame_));
```

Comment:
One minor thing here, you should be able to modify the value `1.0` using a ROS parameter as [AMCL does here](http://wiki.ros.org/amcl#Parameters) with `transform_tolerance`: https://github.com/ros-planning/navigation/blob/melodic-devel/amcl/src/amcl_node.cpp#L442

Reply:
Done, I've added an env var for it. I was doubting whether to leave 0 as its default value (then it will preserve the current behavior for Hector SLAM) or 1.0, since that assures that move base can be used. I left the latter but please let me know if you have any objection.